code/controllers/configuration/entries/game_options.dm

```diff
@@ -401,3 +401,5 @@
 	integer = FALSE
 
 /datum/config_entry/flag/disallow_circuit_sounds
+
+/datum/config_entry/str_list/lua_path
```

Comment:
game_options are for mostly ic or gameplay settings, this would likely go in its own file here because its its own file in the config folder.

Reply:
```suggestion
/datum/config_entry/str_list/lua_path
	protection = CONFIG_ENTRY_LOCKED
```

Reply:
oh wait, this is a list, you have to also hide it.